definitions/bulk-action/json/Jurisdiction.json

```diff
@@ -3,6 +3,7 @@
     "LiveFrom": "01/01/2017",
     "ID": "DIVORCE",
     "Name": "Family Divorce",
-    "Description": "Family Divorce: dissolution of marriage"
+    "Description": "Family Divorce: dissolution of marriage",
+    "Shuttered": "True"
```

Comment:
I can see this is defined differently in different repos, but I'd prefer 

```
    "Shuttered": true
```

https://github.com/hmcts/unspec-service/blob/af7f826ac0694bfac51f8d3cd7faed9c35056de2/ccd-definition/Jurisdiction.json

Reply:
boolean is boolean and not string :)

Reply:
Yep, I did it that way to follow the reference implementation guide, but if ccd config can handle boolean rather than string then I agree that that's preferable.

This PR is low priority at the moment as we've been waiting a little while to get the word to merge this but I'll update and re-test this after code freeze 🙂